app/views/idv/in_person/ready_to_verify/show.html.erb

```diff
@@ -108,10 +108,6 @@
         <%= t('date.day_names')[5] %>: <%= @presenter.selected_location_hours(:saturday) %><br>
         <%= t('date.day_names')[6] %>: <%= @presenter.selected_location_hours(:sunday) %>
       </div>
-      <div>
-        <span class="usa-sr-only"><%= t('in_person_proofing.body.barcode.retail_phone_label') %>: </span>
-        <%= @presenter.selected_location_details['phone'] %>
```

Comment:
since we're removing this we also want to update the presenter spec to remove the test for the phone details 
https://github.com/18F/identity-idp/blob/main/spec/presenters/idv/in_person/ready_to_verify_presenter_spec.rb#L53

Reply:
I think we should also be able to remove phone from the pilot file 
https://github.com/18F/identity-idp/blob/main/config/ipp_pilot_usps_facilities.json

Reply:
https://github.com/18F/identity-idp/blob/bd5d12b9becabc663546f96d929461dc13af2d68/app/services/usps_in_person_proofing/post_office.rb#L7

Reply:
https://github.com/18F/identity-idp/blob/bd5d12b9becabc663546f96d929461dc13af2d68/app/services/usps_in_person_proofing/proofer.rb#L217

Reply:
I'm gonna remove it from the struct file too. I don't want it to be in one place only and be confusing as to why it's there and no where else.